src/Tests/dotnet-MsiInstallation.Tests/README.md

```diff
@@ -0,0 +1,73 @@
+![Image of Hyper-V Manager showing switch manager dialog](images/virtual-switch-manager.png)
+
+
+Create a new Virtual Switch connected to your external network adapter, and check the box that says "Allow management operating system to share this network adapter."
+
+![Image showing creation of a network switch](images/network-switch-creation.png)
+
+- Create a Hyper-V Virtual machine. The simplest way is to use 'Quick Create' and modify the settings after creation -- it will find an appropriate ISO for you. If you need a more specific build, you can follow the steps at https://learn.microsoft.com/virtualization/hyper-v-on-windows/quick-start/create-virtual-machine. Either way, apply the following modifications:
+  - You will need to choose a name for the virtual machine (used by the host) and a machine name for it when setting up Windows inside the VM.  You can choose whatever names you want, but "Test VM" and "TestVM" are good defaults.
+  - If needed, you can download a Windows 11 .iso here: https://www.microsoft.com/en-us/software-download/windows11/
+  - In the networking configuration for the VM, select the Virtual Switch you created.
+  - Make sure that you create a generation 2 VM so UEFI is supported (this occurs by default in Quick Create if it is supported on your machine, which needs TPM 2.0.)
+  
+  
+![The generation 2 box should be selected](images/virtual-machine-generation-2.png)
+
+  - Right click the VM and go to settings. Under Security, check "Enable Trusted Platform Module" (and possibly the "Encrypt state..." checkbox under it), which is required to install Windows 11
+  - Start the VM and install Windows
+    - Probably you don't want to sign on to the test VM with a Microsoft account.  Setting up with a local account is tricky, but you can do so with these steps: https://web.archive.org/web/20240120203712/https://www.tomshardware.com/how-to/install-windows-11-without-microsoft-account
+  - In the VM settings in Hyper-V manager, enable all the integration services (so you can copy/paste files to the VM, for example)
+- In network settings inside the VM, switch the network connection type to Private Network, and turn on Network discovery and File and printer sharing.
+- Inside the VM, set the HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System\LocalAccountTokenFilterPolicy registry value to 1 ([reference](https://learn.microsoft.com/en-us/troubleshoot/windows-server/windows-security/user-account-control-and-remote-restriction)).  This will allow you to access the admin share (`\\TestVM\c$`).
+
+You may also run this under an admin prompt:
+`REG ADD HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System /v LocalAccountTokenFilterPolicy /t REG_DWORD /d 1`
+- Browse to the admin share in File Explorer to confirm it's working.  You will need to enter the username and password for the VM.  Select the option to save the login information.  This will allow the tests to access the VM.
```

Comment:
```suggestion
- After all of this, you _must_ restart your host machine and the VM. Failure to do so will result in 'Access Denied.' Browse to the admin share in File Explorer to confirm it's working.  You will need to enter the username and password for the VM.  The username will likely be the PC name\the user name, such as `WINDEV2401EVAL\User`, and you need to manually create a password in the VM. Select the option to save the login information.  This will allow the tests to access the VM.
```